doc : add contribution guidelines

purohitdheeraj opened the pull request:
# Description

I have added contributing guidelines in CONTRIBUTING.md
some keys section added in this commit are
- how to contribute (fork, pull request)
- basic info on communication, issues, attribution

Fixes #22 

## Type of change

- [x] 📝 Documentation Update

## Added tests?
- [x] 🙅 no, because they aren't needed

ekas commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,79 @@
+10. **Merge**: Once your pull request is approved, it will be merged into the main branch.
+
+## Coding Standards
+
```
Add more points here:

- Add Unit Test Cases
- Add text tokens for Internationalization (for this also add check box for PR template)
- Try to reuse existing components, if you don't find any existing please make one wherever necessary. 
- Please don't leave any Debugger & Console.log statements.
- Please comment your code for other developers where you feel logic is complex.
- Try to write code which is easily understandable by other fellow developers.

ekas commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,79 @@
+
+## Coding Standards
+
+Please follow the coding standards and guidelines specific to the "JSLovers" project. If you're unsure about any coding standards, you can check the project's `CONTRIBUTING.md` or ask for clarification in your pull request or by reaching out to maintainers.
```
```suggestion
Please follow the coding standards and guidelines specific to the "JSLovers" project. If you're unsure about any coding standards, or ask for clarification in your pull request or by reaching out to maintainers.
```

And this line to readme and create new section there with link to contributing guide.

ekas commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,79 @@
+
+## Communication
+
+If you have questions or need assistance, you can reach out to the project maintainers through GitHub issues or other communication channels, such as a dedicated Discord server or Slack workspace (if applicable).
```
Add the Discord link here or pass the readme section link for Discord here.

purohitdheeraj commented:
Hey @ekas thanks for the feedback , based on that i have updated contributing guidelines and added a new section in `README` directing to `CONTRIBUTING`
